code-challenges.js

```diff
@@ -33,6 +36,10 @@
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
+// Create a const "boiling" is equal to the input of "temp"
+// and the input if it is at 212 display "at boiling point"
+// at 350 input is above boiling point
+// below 212 is  below boiling point
```

Comment:
Pseudocode should be the step-by-step process that you take to solve the problem not just restating the initial prompt.  This should include vocabulary and any logic that you will utilize such as conditionals, built-in methods, or iteration.